android/assets/i18n/messages.properties

```diff
@@ -38,30 +38,30 @@ achievements.title=Achievements
 achievements.unlock-requirements={0} to unlock
 
 custom-songs.title=Custom Songs
-custom-songs.description=Want a level that matches your favourite song?\n\nCopy an MP3 file to the folder below and we'll generate a level just for you!
+custom-songs.description=Want a level that matches your favourite song?\n\nCopy an MP3 file to the folder below to generate a level just for you!
 custom-songs.copy-folder=Copy folder path
 custom-songs.view-folder=Open folder
 
 level-select.title=Level Select
 level-select.more-coming-soon.title=More coming soon
-level-select.more-coming-soon.ask-for-recommendations=Have any recommendations for great, freely licensed music? Suggest it on GitHub.
-level-select.more-coming-soon.when-we-find-time=When we find the time to add more songs, we'll make sure to check out the list of suggestions.
+level-select.more-coming-soon.ask-for-recommendations=Suggest freely licensed music you like on GitHub.
+level-select.more-coming-soon.when-we-find-time=More songs will be added from the list of suggestions when we find the time.
```

Comment:
This is great. I love how it is far more concise, but still keeps the casual tone.